InvenTree/InvenTree/celery.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from celery import Celery
+
+# set the default Django settings module for the 'celery' program.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'InvenTree.settings')
+
+app = Celery('InvenTree')
+
+# Using a string here means the worker doesn't have to serialize
+# the configuration object to child processes.
+# - namespace='CELERY' means all celery-related configuration keys
+#   should have a `CELERY_` prefix.
+app.config_from_object('django.conf:settings', namespace='CELERY')
+
+# Load task modules from all registered Django app configs.
+app.autodiscover_tasks()
+
+
+@app.task(bind=True)
```

Comment:
Tasks should have their own dedicated `tasks.py`. I know this is from Celery "get started" doc but just as a reminder.

Reply:
Yes this it is just a dummy task to show that everything is configured correctly. I will move it to a separate file.